manifests/init.pp

```diff
@@ -45,6 +45,12 @@
   Array[Cron::User]    $users_deny         = [],
   Boolean              $manage_users_allow = false,
   Boolean              $manage_users_deny  = false,
+  Boolean              $manage_crontab     = false,
+  String               $crontab_shell      = '/bin/bash',
```

Comment:
can we use Stdlib::Absolutepath here?
```suggestion
  Stdlib::Absolutepath $crontab_shell      = '/bin/bash',
```

Reply:
This change was made, had to add dependency on stdlib as this module didn't already depend on that module.